app/models/split.rb

```diff
@@ -143,6 +144,18 @@ def registry
     end
   end
 
+  def require_app_name_prefix=(value)
+    if instance_variable_defined?(:@require_app_name_prefix)
+      @require_app_name_prefix
+    else
+      @require_app_name_prefix = ActiveRecord::Type::Boolean.new.cast(value)
+    end
+  end
```

Comment:
I think you can be fancy with the new APIs here:

```ruby
attribute : require_app_name_prefix, :boolean
```

Should give you the predicate method for free too.

Reply:
There are several instances of doing it this way in the code already, so I'd probably want to change them all, which I sort of didn't want to uncork :)

Reply:
As long as it's in your mental backlog 🤷‍♂️

Reply:
I _will_ call out one of my favorite takeaways from last year's RailsConf:
https://twitter.com/sarahmei/status/1084980571019796481

But then I'll move onto the rest of the PR 😉

Reply:
I think I’d likely only change if they deprecated this API :) my brain is not large enough to fit a refactoring backlog in it. :)